src/plotter/histo.py

```diff
@@ -24,62 +25,63 @@ def __init__(
         self,
         title: str,
         th: TH1,
-        lineColor: int = ROOT.kBlack,
-        fillColor: Optional[int] = None,
-        drawOption: str = "",
+        linecolor: int = ROOT.kBlack,
+        fillcolor: Optional[int] = 0,
+        drawoption: str = "",
         configPath: str = "",
     ) -> None:
         """
         Arguments:
             th (``TH1``): ROOT histogram
-            lineColor (``int``): color of the histogram line
-            fillColor (``int/None``): color of the histogram fill,
+            linecolor (``int``): color of the histogram line
+            fillcolor (``int/None``): color of the histogram fill,
                 can be None
-        """
-        self.th = th
+        """    
+        self.th = th    
         self.title = title
-        self.lineColor = lineColor
-        self.fillColor = fillColor
+        super().__init__()
+        
+        self.linecolor = linecolor
+        self.fillcolor = fillcolor
         self.config = loader.load_config(configPath) if configPath != "" else {}
         self.apply_all_style()
-        if drawOption != "":
-            self.drawOption = drawOption
-
+        if drawoption != "":
+            self.drawoption = drawoption
+            
         self.isTH1 = th.InheritsFrom("TH1")
         self.isTGraph = th.InheritsFrom("TGraph")
-
+        
```

Comment:
```suggestion

```
vscode has `trim trailing whitespace` option. Probably better to only enable per workspace (in some project in can introduce 100s of changes a people do not like that ...). Alternatively there is a command.